.circleci/config.yml

```
@@ -66,6 +66,37 @@ jobs:
             - "deps"
             - "_build"
       - run: mix test --only conformance
+  dialyzer:
+    working_directory: ~/ex_plasma
+    docker:
+      - image: circleci/elixir:1.9.2
+    environment:
+      MIX_ENV=dev
+    steps:
+      - checkout
+      - run: mix local.rebar --force && mix local.hex --force
+      - restore_cache:
+          key: v1-mix-cache-{{ .Branch }}-{{ checksum "mix.lock" }}
+      - restore_cache:
+          key: v1-dialyzer-deps-cache-{{ .Branch }}-{{ checksum "mix.lock" }}
```

```suggestion
          key: v1-dialyzer-plt-cache-{{ .Branch }}-{{ checksum "mix.lock" }}
```

got confused for a moment whether it's restoring deps or plts